web/src/pages/api/v1/oidc/openid-configuration.ts

```diff
@@ -29,6 +29,8 @@ export default async function handleOidcConfig(
     authorization_endpoint: `${OIDC_BASE_URL}/authorize`,
     grant_types_supported: ["authorization_code", "implicit"],
     service_documentation: "https://docs.worldcoin.org/id/sign-in",
+    op_policy_uri: "https://developer.worldcoin.org/privacy-statement",
```

Comment:
we need to add the redirects from here to the respective Pactsafe documents

Reply:
Done